# HG changeset patch
# User jwe
# Date 832737853 0
#      Wed May 22 04:04:13 1996 +0000
# Node ID d37f8673180684e2b40aad4da0f0f97731fbc3f1
# Parent  e759180dd7db7959c8a988c50ce253ee3f14d65c
[project @ 1996-05-22 04:04:13 by jwe]
Initial revision

diff --git a/src/oct.h b/src/oct.h
new file mode 100644
--- /dev/null
+++ b/src/oct.h
@@ -0,0 +1,48 @@
+/*
+
+Copyright (C) 1996 John W. Eaton
+
+This file is part of Octave.
+
+Octave is free software; you can redistribute it and/or modify it
+under the terms of the GNU General Public License as published by the
+Free Software Foundation; either version 2, or (at your option) any
+later version.
+
+Octave is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with Octave; see the file COPYING.  If not, write to the Free
+Software Foundation, 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+
+*/
+
+#if !defined (octave_oct_h)
+#define octave_oct_h 1
+
+// Things that are often included to create .oct files.
+
+// config.h needs to be first because it includes #defines that can */
+// affect other header files.
+
+#include <octave/config.h>
+
+#include <octave/defun-dld.h>
+#include <octave/error.h>
+#include <octave/gripes.h>
+#include <octave/help.h>
+#include <octave/mappers.h>
+#include <octave/oct-obj.h>
+#include <octave/oct-obj.h>
+#include <octave/utils.h>
+
+#endif
+
+/*
+;;; Local Variables: ***
+;;; mode: C++ ***
+;;; End: ***
+*/
